tool/flutter_ci_script_stable.sh

```diff
@@ -14,6 +14,8 @@ declare -ar PROJECT_NAMES=(
     "add_to_app/multiple_flutters/multiple_flutters_module"
     "add_to_app/plugin/flutter_module_using_plugin"
     "add_to_app/prebuilt_module/flutter_module"
+    # TODO(ewindmill): Already updated for 3.22, breaks CI due to using WidgetStateProperty and friends
+    # "ai_recipe_generation"
```

Comment:
We need to hold off landing this until this carve out is not required. Only things in `experimental` are allowed to ignore `stable` CI =)